packages/server/package.json

```diff
@@ -35,6 +36,8 @@
     "@types/rimraf": "3.0.0",
     "@types/websocket": "^1.0.0",
     "abort-controller": "^3.0.0",
+    "eslint": "6.8.0",
+    "eslint-plugin-bigtest": "^0.1.0",
```

Comment:
Shouldn't these be runtime dependencies? How will the arrive onto the end-users system?

Reply:
definitely.  Moved